.github/workflows/ci.yml

```diff
@@ -33,7 +34,20 @@ jobs:
         with:
           sh_checker_comment: false
           sh_checker_exclude: "src"
+
+  try-run-fuelup-init:
+    needs: cancel-previous-runs
+    name: Try fuelup installation with fuelup-init
+    strategy:
+      matrix:
+        job:
+          - os: ubuntu-latest
+          - os: macos-latest
+    runs-on: ${{ matrix.job.os }}
+    steps:
+      - uses: actions/checkout@v3
       - name: Attempt to install fuelup through fuelup-init.sh
+        if: github.event_name != 'release' || github.event.action != 'published'
```

Comment:
There's only a single step here, so maybe move this `if` to be for the whole job?